pyproject.toml

```diff
@@ -1,3 +1,9 @@
 [tool.black]
 line-length = 100
 
+[tool.pytest.ini_options]
+markers = [
+    "integration",
+    "model",
+    "team",
+]
```

Comment:
Can we revert this change? It seems to be causing pytest cache management issues when running tests, possibly because pytest may think this is the root folder.

Reply:
Yes, feel free to revert as needed